.travis.yml

```diff
@@ -7,7 +7,6 @@ env:
   global:
     - ROS_DISTRO="indigo"
     - UPSTREAM_WORKSPACE=file
-    - CATKIN_CONFIG='--no-install'
```

Comment:
without this option travis is not able to compile this repo, please don't remove it